examples/tests/test_03_message_provider.py

```diff
@@ -0,0 +1,59 @@
+"""
+Test Message Pact provider.
+
+Unlike the standard Pact, which is designed for HTTP interactions, the Message
+Pact is designed for non-HTTP interactions. This example demonstrates how to use
+the Message Pact to test whether a provider generates the correct messages.
+
+In such examples, Pact simply checks the kind of messages produced. The consumer
+need not send back a message, and any sideffects of the message must be verified
+separately.
+
+The below example verifies that the consumer makes the correct filesystem calls
+when it receives a message to read or write a file. The calls themselves are
+mocked out so as to avoid actually writing to the filesystem.
```

Comment:
>The below example verifies that the consumer makes the correct filesystem calls
when it receives a message to read or write a file. The calls themselves are
mocked out so as to avoid actually writing to the filesystem.

Is this the right comment for this file? Should this be in the consumer side?

Reply:
Good catch... bad copy-pasta on my end 😆